orchestrator-salt/src/main/resources/salt/salt/postgresql/scripts/backup_db.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+if [ $# -ne 6 ]; then
+  echo "Invalid inputs provided"
+  echo "Script accepts five inputs:"
+  echo "  1. Cloud Provider (azure | aws)"
+  echo "  2. PostgreSQL host name."
+  echo "  3. PostgreSQL port."
+  echo "  4. Object Storage Service url to place backups."
+  echo "  5. PostgreSQL user name."
+  echo "  6. PostgreSQL password."
+  exit 1
+fi
+
+CLOUD_PROVIDER="$1"
+HOST="$2"
+PORT="$3"
+BACKUP_LOCATION="$4"
+USERNAME="$5"
+export PGPASSWORD="$6" # We can provide the password to pg_dump through this variable, or in ~/.pgpass
+
+BACKUP_DATE=$(date '+%Y-%m-%dT%H:%M:%SZ')
```

Comment:
The date directory part of the backup location will be coming from the datalake DR service calls, since we need to sync that across the different backup types. So you don't need to generate it dynamically in the the scripts.

Reply:
Done, but I'm still generating a date for the temp files that Azure must create before uploading to blob storage -- if you notice the `$(date '+%Y-%m-%dT%H:%M:%SZ')` still hanging around.